src/blog/kuadrant-v1.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Kuadrant v1
+date: 2024-11-28
+author: Jason Madigan
+---
+
+**Update:** Kuadrant [v1.0.1](https://github.com/Kuadrant/kuadrant-operator/releases/tag/v1.0.1) is now available, with some small bugfixes.
+
+### Kuadrant v1!
+
+We're thrilled to announce the release of Kuadrant [v1.0.0](https://github.com/Kuadrant/kuadrant-operator/releases/tag/v1.0.0)! This marks a major milestone in the Kuadrant project after its recent acceptance into [CNCF's sandbox](https://www.cncf.io/sandbox-projects/).
+
+### What is Kuadrant?
+
+Kuadrant is a set of Kubernetes-native controllers, services, and APIs that provide gateway policies for existing [Gateway API](https://gateway-api.sigs.k8s.io/) providers in both single and multi-cluster environments. It builds on top of Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/) and technologies such as Istio and Envoy to introduce provider-agnostic Gateway Policies for Kubernetes.
+
+### Why v1 Matters
+
+Reaching v1 signifies our project's stability, maturity, and production-readiness. With v1, we believe Kuadrant is now ready to power key workloads in Kubernetes environments.
+
+### What's new in v1?
+
+There are too many features and bug fixes to mention (although you are welcome to peruse our [release changelogs](https://github.com/Kuadrant/kuadrant-operator/releases/tag/v1.0.0)). Here are some highlights:
```

Comment:
Given that this is the 1.0 release, this feels like a HUGE missed opportunity.  You'll never have the chance to advertise these features again.  Take a look at Kubernetes release blogs for an example of how you should doc features for a major release: https://kubernetes.io/blog/2024/08/13/kubernetes-v1-31-release/

Reply:
Yeah I think that's fair, and I've always liked how Kubernetes and other projects highlight big ticket features. Will have a go at expanding this notes set a bit to highlight new features better, with links off to further ref material